index/src/main/scala/org/dbpedia/spotlight/db/CreateSpotlightModel.scala

```diff
@@ -245,9 +253,9 @@ object CreateSpotlightModel {
     } else {
       new LanguageIndependentTokenizer(Set[String](), stemmer, locale, tokenStore)
     }
-    val fsaDict = FSASpotter.buildDictionary(sfStore, tokenizer)
+    //val fsaDict = FSASpotter.buildDictionary(sfStore, tokenizer)
```

Comment:
Hey If you leave this commented out, we won't build the data structures for doing the spotting via the SFA. If you uncomment them do things break ?